src/scripts.js

```diff
@@ -103,10 +101,10 @@ function findCheckedBoxes() {
 function findTaggedRecipes(selected) {
   let filteredResults = [];
   selected.forEach(tag => {
-    let allRecipes = recipes.filter(recipe => {
+    let recipeData = recipes.filter(recipe => {
```

Comment:
Should `recipes` be `allRecipes`? Since it looks like you changed the name of that array global variable.

Reply:
Eventually yes. I started to change things over and then it became a painful process. So for now most of the references are using the data file and will have to be migrated over to the API still. This was part of the issue i was mentioning where the API doesn't have the same properties, so it'll be a bit of a process to make sure that all of the array functions are able to access the correct information.